### DISCUSSION TITLE
Support for Azure Cloud using ACA

### DISCUSSION THREAD

**Discussion by vikramjb** [artilleryio/artillery#2511] [Artillery]
Is there plans to support Azure cloud and run the tests on Azure Container Apps ? If it's being worked on I will leave it alone else, I am keen/interested on doing it myself. 

What are your thoughts @hassy @bernardobridge ?

**Comment by hassy** [comment]
hi @vikramjb 👋 We plan to support it eventually, but no ETA yet.

**Reply by vikramjb** [reply]
@hassy I saw one of your videos where you were explaining that support will be extended to Kubernetes and that will help spread this to other cloud providers. Is that the intention still or are you ok If I do some work relating to support "[Azure Container Apps](https://azure.microsoft.com/en-au/products/container-apps)".

This will be the first time I wil be contributing towards an open source project so I will need some hand holding with regards to the standards you follow.

**Reply by JanakiramanPachaiyappan** [reply]
Hi @vikramjb - were you able to achieve this using ACA? Can you throw some light on how you plan to do this or if already done, do you have some examples?

@hassy - would love to hear from you on any updates to this thread.

**Comment by dpjp** [comment]
Following. 
@hassy  - Am also looking for some help to setup and execute artillery - playwright tests in Azure container apps. Would appreciate if you have any updates towards this.

**Comment by hassy** [comment]
It's coming soon (next few months) but no firm ETA yet. Will update here when there's something to try. Feel free to email me direct at hveldstra@artillery.io as well if your team really needs this and you'd be OK to try an early buggy preview.

**Comment by vikramjb** [comment]
@JanakiramanPachaiyappan @hassy @dpjp  👋 

 I was able to run a load test script on the following azure services. A lot manual work was needed but I got it running and I stopped there.

1. Azure Container Instance
2. Azure Container Apps
3. Azure Container Jobs

Theoretically, we should be able to run the load test script on an Azure function as well, although I haven't tried it.  


@hassy I am keen to try on a buggy preview 👍

**Comment by hassy** [comment]
@vikramjb Azure support has been released in Artillery v2.0.19. Docs here: https://docs.art/azure
